### PR TITLE
Add Prettier

### PR DESCRIPTION
**Pull request by jackfranklin**

### CODE REVIEW

**Comment by jackfranklin** [comment]
@rtfeldman the first part of https://github.com/rtfeldman/node-test-runner/issues/310, linting to follow but this first part was super easy :)

**Comment by jackfranklin** [comment]
@rtfeldman I'm not sure why there is a failure on Appveyor (it's saying it cannot find files to run Prettier on) but it works on Travis - might need your help with that!

File: package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
   },
   "scripts": {
     "flow": "flow",
-    "test": "flow check && mocha tests"
+    "test": "flow check && npm run prettier:check && mocha tests",
+    "prettier:check": "prettier 'lib/**/*.js' --list-different",
```
Comment (rtfeldman) [inline package.json]: Looks like the single quotes are the problem: https://github.com/prettier/prettier/issues/4561#issuecomment-392817015

Try it without single quotes:
```suggestion
    "prettier:check": "prettier \"lib/**/*.js\" --list-different",
```

If that works, we should give `prettier:write` the same treatment!

**Comment by rtfeldman** [comment]
Nice, that did the trick!

**Pull request merged**